core/src/test/java/org/elasticsearch/indices/recovery/IndexRecoveryIT.java

```diff
@@ -143,7 +143,6 @@ private void slowDownRecovery(ByteSizeValue shardSize) {
                 .setTransientSettings(Settings.builder()
                                 // one chunk per sec..
                                 .put(RecoverySettings.INDICES_RECOVERY_MAX_BYTES_PER_SEC, chunkSize, ByteSizeUnit.BYTES)
-                                .put(RecoverySettings.INDICES_RECOVERY_FILE_CHUNK_SIZE, chunkSize, ByteSizeUnit.BYTES)
```

Comment:
hmm so how this works is that we take the entire shard size and div it by 10. So this would at max take 10 sec to complete since we share the rate limiter across recovery workers. I also don't see that we are waiting for things to finish. The test seems to finish pretty quickly or at least during the time we throttle? We are also protected by the rate limiter because it won't check for very small chunks (`getMinPauseCheckBytes`) so I think we are good here?